### ISSUE TITLE
Strava service is not downloading lap information

### ISSUE THREAD

**Issue by gutierlf** [cpfair/tapiriik#260]
Strava does currently support laps, contrary to a comment `StravaService#DownloadActivity`. That method needs to be updated to get the information from Strava and add it to the activity.

**Comment by gutierlf** [comment]
I will volunteer to implement this enhancement and send a pull request. Is there any test infrastructure to use along the way?

**Comment by cpfair** [comment]
No, unfortunately not - all I have is a file on my hard drive called `act_test.py` with 3+ years of accrued one-off test scripts, I can't say it was a good decision...

**Comment by gutierlf** [comment]
Coming back to this after a while.

Despite your response that there is no test infrastructure, I found the [tapiriik/testing](https://github.com/cpfair/tapiriik/tree/master/tapiriik/testing) directory which does have some non-trivial content. However, I'm having trouble running the tests there. Are these tests still valid or have they gone stale?

By the way, I've been trying mostly without vagrant, but I tried once with vagrant, and still had trouble.
